Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2015 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura, Mario Carneiro
 -/
+import Aesop
```

Comment:
This is too heavy an import for this file I think, and you don't need it with my fix below.

Reply:
Can you explain more why importing aesop is a heavy import? Those proofs can be replaced by the output of aesop?, right? (In any case, your proposed proof is nicer, so I have removed the import again.)

Reply:
Yes, this makes sense - thanks for explaining. (I checked: aesop has about 15000 lines of code, in a hundred files. That's definitely a heavy dependency.)